### PR TITLE
Typo on client side configure environment in website fixed

### PR DESCRIPTION
**Pull request by shljessie**
@hyochan

### CODE REVIEW

**Review by hyochan — APPROVED** [review]
Wow, thanks for the fixes!! Amazing :100:

**Comment by hyochan** [comment]
I am ready to merge this after you rebase this. Please ask @DevYuns for help if you have trouble!

**Comment by shljessie** [comment]
I am ready to merge @hyochan ! Thanks for helping out @DevYuns ! I'll review and study github well so that this doesn
t happen again :)

File: docs/client/installation.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ Client project belongs to seperate directory [client](https://github.com/doobool
    ```
    cd client
    ```
+   - Also note that yarn.lock and package-lock.json sometimes make collision. Try to delete one of them.
+
 3. Copy the environment file.
    ```
-   cp .config.sample.ts .config.ts
+   cp config.sample.ts .config.ts
```
Comment (hyochan) [inline docs/client/installation.md]:
```
cp config.sample.ts config.ts
```

Please change this to above.

File: docs/client/installation.md [hunk 2]
```diff
@@ -28,3 +30,14 @@ Client project belongs to seperate directory [client](https://github.com/doobool
    ```
    yarn start
    ```
+   - If you have an error in node_modules, try
+   ```
+      rm -rf node_modules && yarn
```
Comment (hyochan) [inline docs/client/installation.md]: <- shift

**Review by hyochan — APPROVED** [review]
LGTM!

**Pull request merged**